apps/sheriff-webservices/package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "@eslint-react/eslint-plugin": "^1.10.1",
     "@eslint/js": "^9.17.0",
-    "@hono/node-server": "^1.12.0",
+    "@hono/node-server": "~1.7.0",
```

Comment:
why?

Reply:
You use hono v3, but @hono/node-server >=1.8 depends on hono v4.